content/applications/inventory_and_mrp/manufacturing/plm/engineering_change_orders.rst

```diff
@@ -0,0 +1,289 @@
+
+.. image:: engineering_change_orders/additional-options-menu.png
+   :align: center
+   :alt: Navigate to the "Add a Step" pop-up by clicking the three horizontal lines icon in tablet
+         view.
+
+In the :guilabel:`Title` field, enter a short step description. Next, in the
+:guilabel:`Instructions` text field, type the instructions of the step in greater detail.
+Optionally, add an image to the :guilabel:`Document` field. Once completed, finish by clicking the
+:guilabel:`Propose Change` button.
+
+.. example::
+   To propose an additional check for broken components, enter the details in the :guilabel:`Add a
+   Step` pop-up window. Doing so creates an instructional quality control point that will be
+   reviewed in the following section.
+
+   .. image:: engineering_change_orders/add-a-step.png
+      :align: center
+      :alt: Fill out the *Add a Step* form to suggest an additional quality control point.
+
+View ECO
+--------
+
+To review the proposed changes, go to the `BOM Changes` ECO type from :menuselection:`PLM app -->
+Overview`, and click on the card to view the contained |ECOs|. Click on the newly created |ECO| in
+the `New` stage, and view a summary of the proposed changes in the :guilabel:`Operation Changes`
+tab. To implement the changes, click the :guilabel:`Revision` smart button.
+
+.. example::
+   An |ECO| adding another check for broken components is created in the `BOM Changes` ECO type
+   found in :menuselection:`PLM app --> Overview`. By default, |ECOs| created from tablet view are
+   named with the manufacturing order number for reference.
+
+   .. image:: engineering_change_orders/view-BOM-change.png
+      :align: center
+      :alt: Find the new ECO in the "BOM Changes" ECO type, in the *New* stage.
+
+On the new |BOM|, switch to the :guilabel:`Operations` tab, and select the :guilabel:`☰ (Show
+Instructions)` icon. Doing so opens a list of :guilabel:`Steps` to perform the operation, with the
+newest instruction titled `New Step Suggestion:`, followed by the user-entered title. Click the line
+item to view the suggested changes.
+
+.. image:: engineering_change_orders/show-instructions.png
+   :align: center
+   :alt: "Show Instructions" icon in the *Operations* tab of a BoM.
+
+On the :ref:`quality control point <quality/quality_control_points>` form, ensure the following form
+fields are accurately filled out to give detailed instructions for operators:
+
+- :guilabel:`Title`: rename to give a concise description of the new instruction
+- :guilabel:`Control per`: using the drop-down menu, determine whether this instruction applies
+  broadly for the :guilabel:`Product`, specifically for this :guilabel:`Operation` *only*, or a
+  particular :guilabel:`Quantity` of the product
+- :guilabel:`Type`: categorizes the control point type. From the drop-down menu, select
+  :guilabel:`Instructions` to detail an instruction for the worker. To receive input from the
```

Comment:
I'm not seeing an Instructions field on my run-through. Also, it may be best to just do a brief breakdown of every field (for instance, I don't see any explanation for the Team or Responsible fields, etc.)

Reply:
This is tough, because this form is technically a quality control point form, and I'm not sure whether I should be exhaustive with the explanation as it's covered in the Quality documentation.  What do you think? Should I still rehash this in the context of PLM?